docusaurus.config.js

```diff
@@ -108,7 +108,7 @@ module.exports = {
       defaultMode: 'dark',
     },
     image: 'https://www.destiny.gg/favicon-196x196.png',
-    metadatas: [
+    metadata: [
```

Comment:
This field was updated to just be "metadata" singular
https://github.com/facebook/docusaurus/pull/5871